<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FillSurfaceClass</class>
 <widget class="QMainWindow" name="FillSurfaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>660</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>678</width>
    <height>568</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FillSurface</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QWidget" name="widget_9" native="true">
      <property name="minimumSize">
       <size>
        <width>660</width>
        <height>210</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>660</width>
        <height>210</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>620</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>620</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="blankWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>620</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>620</width>
           <height>60</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="blank1">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="blank2">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="blank3">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_8" native="true">
         <property name="minimumSize">
          <size>
           <width>620</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>620</width>
           <height>60</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pre">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>上一题</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="next">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>下一题</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
